### DISCUSSION TITLE
Command to show all of the commands without having to open ISE

### DISCUSSION THREAD

**Discussion by hellaAnonymous** [PowerShell/PowerShell#15346] [General]
I suggest that you add a command such as "cmds" or  that will show all of the commands, because, I'm getting tired of opening ISE just to see a list of all the commands. and I bet some other people are too!

**Comment by vexx32** [comment]
`Get-Command` will do this.

**Reply by rkeithhill** [reply]
And if you're on Windows, you can use `Get-Command | Out-GridView` to open a window showing all the commands.
